event-processing/lambda/models/cleansed-event.ts

```diff
@@ -15,6 +15,21 @@ export interface ICleansedEvent {
     extensions?: unknown | undefined;
 }
 
+function CleanseEvidenceEvent(evidence: ICleansedEvidenceEvent) {
```

Comment:
Something I only just noticed.

You can change this function to be a private static method on the Cleansed event class.